src/probes/isUnsafeCallee.js

```diff
@@ -16,13 +15,9 @@ function main(node, options) {
   const { analysis, data: calleeName } = options;
 
   analysis.addWarning("unsafe-stmt", calleeName, node.loc);
-
-  return ProbeSignals.Skip;
 }
 
 export default {
   name: "isUnsafeCallee",
-  validateNode,
-  main,
-  breakOnMatch: false
+  validateNode, main, breakOnMatch: false
 };
```

Comment:
We should keep the original code. No need to remove the skip for me.

Reply:
Note that here we degrade this probe to solve an issue in another (not what we expect). For me we should move isRequire before this probe and then improve the break in isRequire

Reply:
agree i went back to the initial implementation of the probe and just moved the order by putting isRequire first in the  list.